roboteam_ai/src/control/ControlUtils.cpp

```diff
@@ -198,14 +198,27 @@ int ControlUtils::rotateDirection(double currentAngle, double targetAngle){
 }
 
 /// Limits velocity to maximum velocity
-Vector2 ControlUtils::VelocityLimiter(Vector2 vel) {
-    if (vel.length() > rtt::ai::constants::MAX_VEL) {
-        vel = vel.stretchToLength(rtt::ai::constants::MAX_VEL);
+Vector2 ControlUtils::VelocityLimiter(Vector2 vel,double maxVel) {
```

Comment:
why?

Reply:
so it works for any velocity (e.g. you can pass a maximum velocity) (see header file)

Reply:
It defaults to the MAX_VEL